.github/workflows/quality-check.yaml

```diff
@@ -22,7 +22,7 @@ jobs:
     strategy:
       matrix:
           python-version: ["3.7", "3.8", "3.9", "3.10"]
-          tensorflow: ["~=2.5.0", "~=2.6.0", "~=2.7.0", "~=2.8.0", "~=2.9.0", "~=2.10.0", "~=2.11.0", "~=2.12.0"]
+          tensorflow: ["~=2.5.0", "~=2.6.0", "~=2.7.0", "~=2.8.0", "~=2.9.0", "~=2.10.0", "~=2.11.0", "~=2.12.0", "~=2.15.0"]
```

Comment:
do we not want to test on the intermediate values too?

Reply:
Wasn't sure, thought that would be a  lot of combinations. I'll go ahead and it.